lib/pages/getStaticProps/redirects.js

```diff
@@ -47,9 +50,27 @@ pages.forEach(({ route, dataRoute, srcRoute }) => {
     ...previewModeRedirect,
   });
 
-  // Preview mode default locale redirect must precede normal default locale redirect
-  addDefaultLocaleRedirect(redirects)(route, target, previewModeRedirect);
-  addDefaultLocaleRedirect(redirects)(route);
+  if (defaultLocale) {
+    // Preview mode default locale redirect must precede normal default locale redirect
+    // addDefaultLocaleRedirect(redirects)(route, target, previewModeRedirect);
+    // addDefaultLocaleRedirect(redirects)(route);
+    const routePieces = route.split("/");
+    const routeLocale = routePieces[1];
+    if (routeLocale === defaultLocale) {
+
+    const nakedRoute =
+      route === `/${routeLocale}` ? "/" : route.replace(`/${routeLocale}`, "");
+      redirects.push({
+        route: nakedRoute,
+        target,
+        ...previewModeRedirect
+      })
+      redirects.push({
+        route: nakedRoute,
+        target: route,
+      })
+    }
+  }
```

Comment:
Is this part of the PR or is this just temporary debugging of the failing Ubuntu test?